src/modules/cacheClient/cacheClient.service.ts

```diff
@@ -76,9 +76,11 @@ export class CacheClient implements ICacheClient {
     }
 
     /**
-     * @description At the time hanldes only authentication errors. Schedules failed requests and starts authentication
-     * @param error
-     * @returns
+     * @description Interceptor of authentication errors. Schedules failed request and starts authentication
```

Comment:
I think "Queue" describes it better. What do you think?
```suggestion
     * @description Interceptor of authentication errors. Queues failed requests and starts authentication process.
```

Reply:
:+1: `Queues` is more appropriate term for this